sysutils/lima/Portfile

```diff
@@ -36,31 +36,29 @@ checksums           rmd160  7e458abe6e031d2f1a260a9f8eb090510aee714f \
 
 build.cmd           make
 
-patchfiles          patch-Makefile.diff \
-                    patch-main.diff
+patchfiles          patch-Makefile.diff
 
 post-patch {
     reinplace "s|@@VERSION@@|${version}|g" ${worksrcpath}/Makefile
 }
 
-destroot {
-    system -W ${worksrcpath} "make install DEST=${destroot}${prefix}"
+post-build {
+    set limactl ${worksrcpath}/_output/bin/limactl
 
-    # install completion for various shells
-    set bash_completions_dir ${destroot}${prefix}/share/bash-completion/completions
-    set fish_completions_dir ${destroot}${prefix}/share/fish/vendor_completions.d
-    set zsh_completions_dir ${destroot}${prefix}/share/zsh/site-functions
+    set bash_completions_dir ${worksrcpath}/_output/share/bash-completion/completions
+    set fish_completions_dir ${worksrcpath}/_output/share/fish/vendor_completions.d
+    set zsh_completions_dir ${worksrcpath}/_output/share/zsh/site-functions
 
     xinstall -d ${bash_completions_dir} ${fish_completions_dir} ${zsh_completions_dir}
 
-    system "
-        ${destroot}${prefix}/bin/limactl completion bash \
-            > ${bash_completions_dir}/${name}
-        ${destroot}${prefix}/bin/limactl completion fish \
-            > ${fish_completions_dir}/${name}.fish
-        ${destroot}${prefix}/bin/limactl completion zsh \
-            > ${zsh_completions_dir}/_${name}
-    "
+    system "${limactl} completion bash > ${bash_completions_dir}/${name}"
+    system "${limactl} completion fish > ${fish_completions_dir}/${name}.fish"
+    system "${limactl} completion zsh > ${zsh_completions_dir}/_${name}"
+}
+
+destroot {
+    system -W ${worksrcpath} "make install DEST=${destroot}${prefix}"
```

Comment:
Why is this overriding the destroot phase rather than using it normally? The default `destroot.cmd` is already `make`; the default `destroot.target` is already `install`. All you need to add to handle the nonstandard destdir variable is:
```
destroot.destdir DEST=${destroot}${prefix}
```

Reply:
The Go port group actively suppresses the default `destroot` phase, and I'm unaware of another way to reinstate it — see [golang-1.0.tcl](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/golang-1.0.tcl#L441:L451).